### ISSUE TITLE
opnsense-bootstrap fails on AWS EC2 instance

### ISSUE THREAD

**Issue by securityguy** [opnsense/update#83]
Using opnsense-bootstrap on a fresh freeBSD 13 completes the install, reboots, and allows network interface configuration via the AWS serial console. However, the VM then hangs after displaying the HTTPS SHA2 fingerprint:

`Configuring firewall.......done.
Syncing OpenVPN settings...done.
Starting NTP service...done.
Starting Unbound DNS...done.
Generating RRD graphs...done.
>>> Invoking start script 'newwanip'
Reconfiguring IPv4 on ena0
>>> Invoking start script 'freebsd'
Starting rtsold.
>>> Invoking start script 'syslog'
>>> Invoking start script 'dnsbl'
>>> Invoking start script 'carp'
>>> Invoking start script 'cron'
Starting Cron: OK
>>> Invoking start script 'beep'
Root file system: /dev/gpt/rootfs
Thu Jan 12 01:03:30 UTC 2023

*** OPNsense.localdomain: OPNsense 22.7.10_2 (amd64/OpenSSL) ***

 LAN (ena0)      -> v4/DHCP4: 10.99.15.157/20

 HTTPS: SHA256 7D 63 A5 70 0E E0 03 2D 86 E9 C1 EE 62 C2 7C 0A
               43 13 0D 90 47 1D EC 08 98 FE A6 1C 80 0A 74 3D`
               
            
Multiple attempts failed using a VM with 2 interfaces (LAN and WAN), and a VM with one interface and configuring it as WAN or LAN.

The system can be booted into single-user mode, but normal reboots fail as above.

**Comment by fichtner** [comment]
I don't think it fails. The default config used just does not enable the console required. Default is VGA, not sure if AWS EC2 emulates a "serial" console but it would make more sense than VGA...


Cheers,
Franco
